### PR TITLE
Feature/send table UI

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

The non-cryptography changes for #263. We add a table with mocked send data and a link on the wallet pop out. Both these will be behind a `v-if` after approval.

[Screencast from 2023-02-19 13-42-24.webm](https://user-images.githubusercontent.com/12705146/219968413-6a215d7b-bfcc-4dcd-9ab8-b710e9a210c6.webm)

### CODE REVIEW

File: frontend/src/store/wallet.ts
Comment (mds1) [inline frontend/src/store/wallet.ts:540]: This method and `copyAddress` above feel like they should be in `utils.ts` and take a `provider` input, since they aren't otherwise reliant on the wallet store

File: frontend/src/pages/AccountSent.vue
Comment (mds1) [inline frontend/src/pages/AccountSent.vue:22]: These are the same import with different names, let's only keep the `useWalletStore` one

File: frontend/src/pages/AccountSent.vue
Comment (mds1) [inline frontend/src/pages/AccountSent.vue:49]: Wdyt about generating the `formattedRows` array with `data.map` and using that to initialize `sendMetadata`, and removing the try/catch since I don't think we really need that here? Would make this a bit more concise and remove the need to initialize an empty array

File: frontend/src/i18n/locales/en-US.json
Comment (mds1) [inline frontend/src/i18n/locales/en-US.json:295]: This key can be deleted and replaced with "date-sent" (we currently use the received key on mobile view)

File: frontend/src/i18n/locales/en-US.json
Comment (mds1) [inline frontend/src/i18n/locales/en-US.json:311]:
```suggestion
    "view-send-history": "View send history",
```

cc @garyghayrat for translation

File: frontend/src/components/WalletRow.vue
Comment (mds1) [inline frontend/src/components/WalletRow.vue:49]: A few comments here:
1. The whole thing looks like the button, but I can only click the text. Let's make the whole thing clickable
2. Let's add some change on hover, maybe the gold outline like we do when hovering the "Send" and "Receive" boxes

File: frontend/src/components/WalletRow.vue
Comment (mds1) [inline frontend/src/components/WalletRow.vue:158]: Nit, extra line break. Maybe would be removed by `yarn prettier`?

File: frontend/src/components/AccountSendTable.vue
Comment (mds1) [inline frontend/src/components/AccountSendTable.vue:96]: This file name and this should be `AccountSentTable` to be consistent with the `AccountSent` page (makes more sense too since the sends already occurred)

File: frontend/src/components/AccountSendTable.vue
Comment (mds1) [inline frontend/src/components/AccountSendTable.vue:1]: How much of this is similar to the `AccountReceiveTable`? Does it make sense to extract a `BaseSendsTable` component and reuse that in the `AccountReceiveTable` and `AccountSentTable` components?

**Review by mds1 — COMMENTED** [review]
looks great overall!

File: frontend/src/i18n/locales/en-US.json
Comment (garyghayrat) [reply frontend/src/i18n/locales/en-US.json:311]: 查看发送历史

Comment (garyghayrat) [reply frontend/src/i18n/locales/en-US.json]: Fix here https://github.com/ScopeLift/umbra-protocol/pull/465/commits/9121397c9a0f7060b5265addbdd9fcb5d0cd9f41

File: frontend/src/i18n/locales/en-US.json
Comment (garyghayrat) [reply frontend/src/i18n/locales/en-US.json]: deleted here https://github.com/ScopeLift/umbra-protocol/pull/465/commits/9121397c9a0f7060b5265addbdd9fcb5d0cd9f41

File: frontend/src/components/AccountSendTable.vue
Comment (alexkeating) [reply frontend/src/components/AccountSendTable.vue]: I think it is similar but a pared down version. There are probably some things that can be abstracted

Comment (mds1) [reply frontend/src/components/AccountSendTable.vue]: Should we track that in a separate issue or do it here? Hmm or maybe it's not worth abstracting since it's only 2 instances 🤔

Comment (alexkeating) [reply frontend/src/components/AccountSendTable.vue]: I think a separate issue would make sense. And we abstract when the next use case comes

File: frontend/src/components/AccountReceiveTable.vue
Comment (mds1) [inline frontend/src/components/AccountReceiveTable.vue:100]: _(arbitrary comment location to start a thread)_

> I like the link in the wallet area, only thing I would say is it might also be good to include a link to it on the send form itself for better discoverability

File: frontend/src/components/AccountReceiveTable.vue
Comment (mds1) [inline frontend/src/components/AccountReceiveTable.vue:121]: _(arbitrary comment location to start a thread)_

> I think we should include some copy somewhere (maybe below the title, or maybe in an (i)-button mouseover tooltip thing) that explains the send history is only for this device, is stored locally, and can't be found by looking on-chain

Comment (alexkeating) [reply frontend/src/components/AccountReceiveTable.vue:121]: Added an i tootip

File: frontend/src/components/AccountReceiveTable.vue
Comment (alexkeating) [reply frontend/src/components/AccountReceiveTable.vue:100]: Added link to sent page on the send page

Comment (mds1) [reply frontend/src/components/AccountReceiveTable.vue:100]: LGTM, might do them as two outline buttons side by side, but ok with doing that in a follow up PR to avoid churn/blocking this, as long as @apbendi approves

File: frontend/src/components/AccountReceiveTable.vue
Comment (mds1) [reply frontend/src/components/AccountReceiveTable.vue:121]: Can we put it next to the "Send Funds" table header instead of the page header, i.e. the highlighted text here? 
image

Comment (alexkeating) [reply frontend/src/components/AccountReceiveTable.vue:121]: Yes will move it now

**Comment by alexkeating** [comment]
This should be good now, pending a successful build

**Review by mds1 — APPROVED** [review]
LGTM, pending @apbendi's review + feature flag to disable this before merging

**Review by apbendi — APPROVED** [review]
Looks good to me! Nice work @alexkeating

**Comment by alexkeating** [comment]
Added feature flags

File: frontend/src/components/AccountReceiveTable.vue
Comment (garyghayrat) [inline frontend/src/components/AccountReceiveTable.vue:97]: Getting this on the Send page 
image

File: frontend/src/components/AccountReceiveTable.vue
Comment (garyghayrat) [inline frontend/src/components/AccountReceiveTable.vue:98]: Getting these on the Receive page
image

Comment (garyghayrat) [reply frontend/src/components/AccountReceiveTable.vue:98]: Above image is from Chrome on Goerli, this one is from Firefox on Goerli:
image

Comment (garyghayrat) [reply frontend/src/components/AccountReceiveTable.vue:98]: I must be doing something wrong cause I don't even see the Send History button under my account 
image

File: frontend/src/components/AccountReceiveTable.vue
Comment (garyghayrat) [reply frontend/src/components/AccountReceiveTable.vue:97]: image

File: frontend/src/components/AccountReceiveTable.vue
Comment (alexkeating) [reply frontend/src/components/AccountReceiveTable.vue:98]: Yea, the routes change seems to be the issue. I am going to drop a message in Element

File: frontend/src/components/AccountReceiveTable.vue
Comment (garyghayrat) [reply frontend/src/components/AccountReceiveTable.vue:97]: I'm no longer encountering this issue

File: frontend/src/components/AccountReceiveTable.vue
Comment (garyghayrat) [reply frontend/src/components/AccountReceiveTable.vue:98]: This is still happening to me @alexkeating on the Receive page while on Chrome, using Metamask Goerli network.

Comment (garyghayrat) [reply frontend/src/components/AccountReceiveTable.vue:98]: image

Comment (alexkeating) [reply frontend/src/components/AccountReceiveTable.vue:99]: I think I fixed this

**Review by garyghayrat — APPROVED** [review]

**Pull request merged**
